modules/reranker-cohere/clients/ranker_meta.go

```diff
@@ -0,0 +1,45 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package clients
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+func (s *client) MetaInfo() (map[string]interface{}, error) {
```

Comment:
there is no `meta` endpoint, this part should be removed. Look [here](https://github.com/weaviate/weaviate/blob/master/modules/generative-cohere/clients/cohere_meta.go) how it should be implemented

Reply:
Ah fantastic, thanks for the clear example!